packages/react-components/src/organisms/SharedResearchCard.tsx

```diff
@@ -50,7 +50,14 @@ const typeStyles = css({
 
 type SharedResearchCardProps = Pick<
   ResearchOutputResponse,
-  'id' | 'created' | 'publishDate' | 'team' | 'title' | 'type' | 'link'
+  | 'id'
+  | 'created'
+  | 'publishDate'
```

Comment:
```suggestion
```

and change to use added date in the code below

Reply:
I guess I should only change this once the FE changes the copy

Reply:
Copy change is already happening in another PR